get_molecule_xyz/get_molecule_xyz.py

```diff
@@ -46,8 +57,14 @@ def cascsv_to_xyz(csv_filename:str,xyz_file_column_label:str,cas_column_label:st
     Returns
     -------
         None
-        Creates an xyz file for each molecule in a csv it can be found 
+        Creates an xyz file for each molecule in a csv it can be found. Also modifies csv fead in. 
     """
+    df = pandas.read_csv(csv_filename)
```

Comment:
this assumes that the files are in the same directory, but the documentation doesn't indicate this